pkg/integration/tests/branch/delete.go

```diff
@@ -31,13 +39,62 @@ var Delete = NewIntegrationTest(NewIntegrationTestArgs{
 			SelectNextItem().
 			Press(keys.Universal.Remove).
 			Tap(func() {
-				t.ExpectPopup().Confirmation().
-					Title(Equals("Delete branch")).
-					Content(Contains("Are you sure you want to delete the branch 'branch-one'?")).
+				t.ExpectPopup().
+					Menu().
+					Title(Equals("Delete branch 'branch-three'?")).
+					Select(Equals("l Delete local branch")).
+					Confirm()
+			}).
+			Tap(func() {
+				t.ExpectPopup().
+					Confirmation().
+					Title(Equals("Force delete branch")).
+					Content(Equals("'branch-three' is not fully merged. Are you sure you want to delete it?")).
+					Confirm()
+			}).
+			Lines(
+				MatchesRegexp(`\*.*branch-four`),
+				MatchesRegexp(`branch-two`).IsSelected(),
+				MatchesRegexp(`branch-one`),
+				MatchesRegexp(`master`),
+			).
+			Press(keys.Universal.Remove).
+			Tap(func() {
+				t.ExpectPopup().
+					Menu().
+					Title(Equals("Delete branch 'branch-two'?")).
+					Select(Equals("r Delete remote branch")).
+					Confirm()
+			}).
+			Lines(
+				MatchesRegexp(`\*.*branch-four`),
+				MatchesRegexp(`branch-two \(upstream gone\)`).IsSelected(),
+				MatchesRegexp(`branch-one`),
+				MatchesRegexp(`master`),
+			).
+			Press(keys.Universal.Remove).
+			Tap(func() {
+				t.ExpectPopup().
+					Menu().
+					Title(Equals("Delete branch 'branch-two'?")).
+					Select(Equals("l Delete local branch")).
+					Confirm()
+			}).
+			Lines(
+				MatchesRegexp(`\*.*branch-four`),
+				MatchesRegexp(`branch-one`).IsSelected(),
+				MatchesRegexp(`master`),
+			).
+			Press(keys.Universal.Remove).
+			Tap(func() {
+				t.ExpectPopup().
+					Menu().
+					Title(Equals("Delete branch 'branch-one'?")).
+					Select(Equals("d Delete local and remote branch")).
```

Comment:
Can we also test that the remote branches view no longer shows the remote branch?